lib/Pithub/Repos.pm

```diff
@@ -85,6 +85,79 @@ sub branches {
     );
 }
 
+=method rename_branch
+
+=over
+
+=item *
+
+Rename a branch
+
+    POST /repos/:user/:repo/branches/:branch/rename
+
+Examples:
+
+    my $b = Pithub::Repos->new;
+    my $result = $b->rename_branch(
+        user => 'plu',
+        repo => 'Pithub',
+        branch  => 'travis',
+        data => { new_name => 'travis-ci' }
+    );
+
+=back
+
+=cut
+
+sub rename_branch {
+    my ( $self, %args ) = @_;
+    croak 'Missing parameters: data' unless $args{data};
+    croak 'Missing parameters: branch' unless $args{branch};
+    croak 'Missing key in parameters: new_name' unless $args{data}->{new_name};
+    $self->_validate_user_repo_args( \%args );
+    return $self->request(
+        method => 'POST',
+        path   => sprintf( '/repos/%s/%s/branches/%s/rename', delete $args{user}, delete $args{repo}, delete $args{branch} ),
+        %args, # Should not contain old name (keep the delete above for $args{data}->{branch})
+    );
+}
+
+=method merge_branch
+
+=over
+
+=item *
+
+Merge a branch
+
+    POST /repos/:user/:repo/merges
+
+Examples:
+
+    my $b = Pithub::Repos->new;
+    my $result = $b->rename_branch(
+        user => 'plu',
+        repo => 'Pithub',
+        data => { base => 'master', head => 'travis', message => 'My commit message' }
+    );
+
+=back
+
+=cut
+
+sub merge_branch {
+    my ( $self, %args ) = @_;
+    croak 'Missing parameters: data' unless $args{data};
+    croak 'Missing key in parameters: base' unless $args{data}->{base};
+    croak 'Missing key in parameters: head' unless $args{data}->{head};
```

Comment:
Could you double check the `croak` messages here to make sure they're in line with the rest of the API?

Reply:
I aligned with other :+1: